data/sql_updates/communication_cost.sql

```diff
@@ -3,26 +3,30 @@ create materialized view ofec_communication_cost_mv_tmp as
 select
     row_number() over () as idx,
     *,
+    s_o_cand_id as cand_id,
+    org_id as cmte_id,
     report_pdf_url(image_num) as pdf_url
-from communication_costs_vw
-where rpt_yr >= :START_YEAR
+from nml_form_76
+where extract(year from communication_dt)::integer >= :START_YEAR
 ;
 
 create unique index on ofec_communication_cost_mv_tmp (idx);
 
 create index on ofec_communication_cost_mv_tmp (cmte_id);
 create index on ofec_communication_cost_mv_tmp (cand_id);
-create index on ofec_communication_cost_mv_tmp (cand_office_st);
-create index on ofec_communication_cost_mv_tmp (cand_office_district);
-create index on ofec_communication_cost_mv_tmp (cand_office);
-create index on ofec_communication_cost_mv_tmp (cand_pty_affiliation);
-create index on ofec_communication_cost_mv_tmp (transaction_dt);
-create index on ofec_communication_cost_mv_tmp (transaction_amt);
+create index on ofec_communication_cost_mv_tmp (s_o_cand_office_st);
+create index on ofec_communication_cost_mv_tmp (s_o_cand_office_district);
+create index on ofec_communication_cost_mv_tmp (s_o_cand_office);
+create index on ofec_communication_cost_mv_tmp (s_o_ind);
+--create index on ofec_communication_cost_mv_tmp (cand_pty_affiliation);
+--create index on ofec_communication_cost_mv_tmp (transaction_dt);
+--create index on ofec_communication_cost_mv_tmp (transaction_amt);
+create index on ofec_communication_cost_mv_tmp (communication_dt);
+create index on ofec_communication_cost_mv_tmp (communication_cost);
```

Comment:
Ok I think for this file the references will have to stay as communication_dt and _cost.  If you look in costs.py  we are aliasing the columns so externally it appears that those columns never went away.  Otherwise I believe the column would have to renamed in the database.